docs/source/models/causes/neonatal/neonatal_encephalopathy.rst

```diff
@@ -146,34 +146,34 @@ Note that these probabilities are not used directly in the model and are include
     * - Symbol
       - Name
       - Definition
-    * - mr
+    * - csmrisk
       - encephalopathy mortality risk
       - The probability that a simulant who was born alive dies from this cause during the neonatal period
 
 
 Modeling Strategy
 +++++++++++++++++
 
-The Neonatal Encephalopathy submodel requires only the birth-weight- and gestation-age-stratified cause specific mortality rates for sepsis during the early and late neonatal periods.
+The Neonatal Encephalopathy submodel requires only the birth-weight- and gestation-age-stratified cause specific mortality risk during the early and late neonatal periods.
 
-The way these CSMRs are used is the same for all subcauses, and therefore is included in the :ref:`Overall Neonatal Disorders Model <2021_cause_neonatal_disorders_mncnh>` page.  This page describes the birth-weight- and gestational-age-specific cause specific mortality rates that are used for this cause on that page, :math:`\text{CSMR}^{\text{encephalopathy}}_{\text{BW},\text{GA}}`.
+The way these CSMRisks are used is the same for all subcauses, and therefore is included in the :ref:`Overall Neonatal Disorders Model <2021_cause_neonatal_disorders_mncnh>` page.  This page describes the birth-weight- and gestational-age-specific cause specific mortality risks that are used for this cause on that page, :math:`\text{CSMR}^{\text{encephalopathy}}_{\text{BW},\text{GA}}`.
 The formula is:
 
 .. math::
     \begin{align*}
-    \text{CSMR}_{\text{BW},\text{GA}}
+    \text{CSMRisk}_{\text{BW},\text{GA}}
     &=
-    \text{CSMR} \cdot \text{RR}_{\text{BW},\text{GA}} \cdot Z
+    \text{CSMRisk} \cdot \text{RR}_{\text{BW},\text{GA}} \cdot Z
     \end{align*}
 
 where 
-:math:`\text{CSMR}` is the cause-specific mortality rate for encephalopathy,
+:math:`\text{CSMRrisk}` is the cause-specific mortality risk for encephalopathy,
 :math:`\text{RR}_{\text{BW},\text{GA}}` is the relative risk of all-cause mortality for a birth weight of :math:`\text{BW}` and gestational age of :math:`\text{GA}`, and :math:`Z` is a normalizing constant selected so that :math:`\int_{\text{BW}} \int_{\text{GA}} \text{RR}_{\text{BW},\text{GA}} \cdot Z = 1`.
 
 .. note::
   the choice to use :math:`\text{RR}_{\text{BW},\text{GA}}` in this equation is essentially arbitrary, and it could be replaced by any other nonnegative "weight function" :math:`w(\text{BW},\text{GA})` as long it doesn't lead to a negative "other causes" mortality hazard.  But with this choice, :math:`Z` is equal to the :math:`1-\text{PAF}` of LBWSG on all-cause mortality.
```

Comment:
I _think_ changing "hazard" to "risk" here makes this into a correct statement... looks like you changed this on the other two cause pages, but missed this one.
```suggestion
  the choice to use :math:`\text{RR}_{\text{BW},\text{GA}}` in this equation is essentially arbitrary, and it could be replaced by any other nonnegative "weight function" :math:`w(\text{BW},\text{GA})` as long it doesn't lead to a negative "other causes" mortality risk.  But with this choice, :math:`Z` is equal to the :math:`1-\text{PAF}` of LBWSG on all-cause mortality.
```